packages/expo-checkbox/src/ExpoCheckbox.tsx

```diff
@@ -1,49 +1,57 @@
 
-const ExpoCheckbox: CheckboxComponent = (props) => {
-  const { color, disabled, onChange, onValueChange, style, value, ...other } = props;
+export default class ExpoCheckbox extends React.PureComponent<CheckboxProps> {
+  /**
+   * @deprecated `Checkbox.isAvailableAsync()` is deprecated and will be removed in future releases.
```

Comment:
Might be worth adding why it is deprecated. Is it because it's now available on all platforms?